rio/Makefile

```diff
@@ -14,7 +14,7 @@ sim:	## Run the simulator
 	python -m robotpy sim
 
 deploy:	## Deploy to the robot (with netconsole)
-	python -m robotpy deploy --nc 
+	python -m robotpy deploy --nc --skip-test --no-install
```

Comment:
this is a little risky, also, you want netconsole, but you don't want the other forms of debugging? i would suggest moving these arguments to a `force-push` target instead of deploy...

Reply:
those have been removed